templates/zerver/api/get-events-from-queue.md

```diff
@@ -77,9 +77,7 @@ endpoint and a queue would be registered in the absence of a `queue_id`.
 
 #### Return values
 
-* `events`: An array (possibly zero-length if `dont_block` is set) of events
-  with IDs newer than `last_event_id`. Event IDs are guaranteed to be increasing,
-  but they are not guaranteed to be consecutive.
+{generate_return_values_table|zulip.yaml|/events:get}
```

Comment:
`events: An array of event objects (possibly zero-length if dont_block is set) of events with IDs newer than last_event_id. Event IDs are guaranteed to be increasing, but they are not guaranteed to be consecutive.`

The `of events` repitition in `zulip.yaml` is a bit weird.  Maybe just `set) with event IDs newer ...` as the edit there?

Reply:
Done.